### PR TITLE
Update throughput schema per ticket #861.

### PR DESCRIPTION
**Pull request by dmggh**

### CODE REVIEW

File: jwst/datamodels/schemas/throughput.schema.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 title: Filter throughput data model
 allOf:
-- $ref: core.schema.yaml
```
Comment (hbushouse) [inline jwst/datamodels/schemas/throughput.schema.yaml]: The core schema should be replaced by referencefile.schema.yaml

Comment (hbushouse) [inline jwst/datamodels/schemas/throughput.schema.yaml]: Also, there was a typo in the mods applied to throughput.py in #715. Line 16 needs to have "ThroughModel" updated to "ThroughputModel". Please include that update here too.

**Comment by hbushouse** [comment]
It appears that throughput.schema.yaml is still missing a reference to "referencefile.schema.yaml". Is that going to be included in another PR?

**Comment by dmggh** [comment]
Yes.

**Pull request merged**